packages/cxl-ui/src/components/cxl-playbook-accordion.js

```diff
@@ -0,0 +1,163 @@
+import { customElement } from 'lit-element';
+import '@conversionxl/cxl-lumo-styles';
+import '@vaadin/vaadin-accordion';
+import '@vaadin/vaadin-checkbox';
+import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
+import cxlAdeftAccordionGlobalStyles from '../styles/global/cxl-playbook-accordion-css.js';
+import { CXLVaadinAccordion } from './cxl-vaadin-accordion';
+
+/**
+ * Allows opening multiple panels simultaneously.
+ * Saves panel state, restores on page load.
+ */
+@customElement('cxl-playbook-accordion')
+export class CxlPlaybookAccordion extends CXLVaadinAccordion {
+  constructor() {
+    super();
+    registerStyles('vaadin-accordion-panel', [CxlPlaybookAccordion._getAccordionPanelStyles()]);
+    registerStyles('vaadin-checkbox', [CxlPlaybookAccordion._getVaadinCheckboxStyles()]);
+  }
+
+  static _getAccordionPanelStyles() {
+    return css`
+      [part='summary'] {
+        box-shadow: 0 4px 12px rgba(36, 38, 40, 0.12);
+        padding: 0;
+      }
+      :host(.emptyContent) [part='toggle'] {
+        display: none;
+      }
+      [part='toggle'] {
+        padding: 12px;
+      }
+    `;
+  }
+
+  static _getVaadinCheckboxStyles() {
+    return css`
+      :host([theme~='custom']) [part='checkbox'] {
+        border: 2px solid !important;
+        width: 18px;
+        height: 18px;
+      }
+      :host([theme~='custom']) [part='label'] {
+        display: none;
+      }
+    `;
+  }
```

Comment:
Why are we doing inline styles in JS suddenly?

Shadow styles should also be SCSS.

Reply:
ok, styles moved to standalone `scss` files https://github.com/conversionxl/aybolit/pull/87/commits/79159a7f4ad17f1265c4e480ebd1bc25933c05e2